### PR TITLE
Rename Transcript to Alternative.

### PR DESCRIPTION
**Pull request by daspecster**
Closer to #2522.

### CODE REVIEW

File: speech/google/cloud/speech/alternative.py [hunk 1]
```diff
@@ -12,11 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Transcript representation for Google Speech API"""
+"""Alternative representation for Google Speech API"""
```
Comment (dhermes) [inline speech/google/cloud/speech/alternative.py]: Can you word this differently?

```
"""Representation of Speech Recognition Alternative for the Google Speech API."""
```

File: speech/google/cloud/speech/alternative.py [hunk 2]
```diff
@@ -12,11 +12,11 @@
...
 
 
-class Transcript(object):
-    """Representation of Speech Transcripts.
+class Alternative(object):
+    """Representation of Speech Alternatives.
```
Comment (dhermes) [inline speech/google/cloud/speech/alternative.py]: plural "Alternatives"?

File: speech/unit_tests/test_alternative.py [hunk 3]
```diff
@@ -15,41 +15,41 @@
 import unittest
 
 
-class TestTranscript(unittest.TestCase):
+class AlternativeTests(unittest.TestCase):
```
Comment (dhermes) [inline speech/unit_tests/test_alternative.py]: s/`AlternativeTests`/`TestAlternative` (this is a general rule of them, name a `TestCase` subclass something like `TestFoo`)

File: speech/google/cloud/speech/__init__.py [hunk 4]
```diff
@@ -18,4 +18,4 @@
 from google.cloud.speech.connection import Connection
 from google.cloud.speech.encoding import Encoding
 from google.cloud.speech.operation import Operation
-from google.cloud.speech.transcript import Transcript
+from google.cloud.speech.alternative import Alternative
```
Comment (dhermes) [inline speech/google/cloud/speech/__init__.py]: Alphabetical?

**Comment by daspecster** [comment]
Updated and squashed.

**Pull request merged**